zanata-war/src/main/java/org/zanata/notification/MessageUnwrapper.java

```diff
@@ -46,7 +46,8 @@ static MessageUnwrapper unwrap(final Message message) {
         ImmutableMap.Builder<String, ? super Object> builder =
                 ImmutableMap.builder();
         for (String propName : propNames) {
-            builder.put(propName, getPropertyValue(message, propName));
+            Object propertyValue = getPropertyValue(message, propName);
+            builder.put(propName, propertyValue == null ? "" : propertyValue);
```

Comment:
This is in the increase test coverage commit as well. After I added the unit test I discovered a bug and fixed it. I probably should separate this one out.